meinberlin/templates/a4modules/module_detail.html

```diff
@@ -48,6 +48,15 @@
                     </li>
                 </ul>
             </nav>
+            {% for phase in module.phases %}
+              {% if phase == module.active_phase %}
```

Comment:
Yes, that's alright. But then it's also shown above the info on the livequestions page with the cta, isn't it? Or only in timeline view? If it's really only shown where we want it to be, you can make one if from the two: `if phase == module.active_phase and live_stream`

Reply:
oh cool! thanks! 
I checked on my current project and it wasn't shown but i'll check timeline view